vault/changelog.early-seed.md

```diff
@@ -2,41 +2,28 @@
 id: 3abd00eb-1c1e-4253-aaf5-dcbe20c21850
 title: Early Seed
 desc: ''
-updated: 1656080391979
+updated: 1656679822589
 created: 1604539200840
 published: true
 nav_exclude: true
 ---
 
 > This page documents the features, enhancements, and fixes in the latest early seed release. The early seed is a preview of the weekly release, so you'll see the next minor version when installing (ex. `0.78.0` instead of `0.77.1`). When dendron ships the general release, the currently installed extension will automatically be updated (but will still have same version).
 
-## 0.101.0
-June 24, 2022
+## 0.102.0
+July 01, 2022
 
-Dendron 0.101 has sprouted 🌱
+Dendron 0.102 has sprouted 🌱
 
-**Depth Filter for Local Graphs:** Want to see the bigger picture? The graph depth filter now lets you control how much context you see in the [[Local Graph View|dendron://dendron.dendron-site/dendron.topic.sidebar.local-graph-view]]
-- More Information: [[Other Filters|dendron://dendron.dendron-site/dendron.topic.graph-view#other-filters]]
-
-**Improved Note Traits:** Rust isn't the only language with traits! We've made a bunch of improvements to traits, including hot reloading, the ability to apply a template, bundling libraries like lodash and luxon, as well as better docs and error handling. 
-- More Information: [[Traits|dendron://dendron.dendron-site/dendron.topic.traits]]
-
-### Features
-- feat(views): view more note links in local graph panel with depth customization (#3072) @joshi ([[docs|dendron://dendron.dendron-site/dendron.topic.graph-view#other-filters]])
+**New Template Variables:** We've added some new template variables like `Current_Quarter` and `Current-Month-Name` to enhance your journaling experience!
```

Comment:
this isn't really a highlight. also variables names should be all upper case  using `_`.

would just not have a highlight for this week